### ISSUE TITLE
Error when trying to configure the plugin

### ISSUE THREAD

**Issue by jdimeo** [DefaultValue/redmine-silent-statuses#2]
I can't access the settings page. Is this related to the fact that I'm using JRuby?

![image](https://user-images.githubusercontent.com/1071438/41362888-806b243c-6f00-11e8-8e18-c80c2f5064f3.png)

**Comment by nowiko** [comment]
@jdimeo 
Have you combated this error? Or it still occurs?

**Comment by jdimeo** [comment]
@nowiko Yes, it is is still a problem. I think it has to do with the fact that we're using a newer version of rails. Instead of:
```
ActionDispatch::Reloader.to_prepare do
  SettingsHelper.send :include, SilentStatusesSettingsHelper
end
```
in `init.rb`, we are doing:
```
include SilentStatusesSettingsHelper
```
and that seems to be working
